docs/sections/user_guide/yaml/tags.rst

```diff
@@ -23,6 +23,24 @@ Or explicit:
 
 Additionally, UW defines the following tags to support use cases not covered by standard tags:
 
+``!datetime``
+^^^^^^^^^^^^^
+
+Converts the tagged node to a Python ``datetime`` object. For example, given ``input.yaml``:
+
+.. code-block:: yaml
+
+   date1: 2024-09-01
+   date2: !datetime "{{ date1 }}"
+
+.. code-block:: text
+
+   % uw config realize -i ../input.yaml --output-format yaml
+   date1: 2024-09-01
+   date2: 2024-09-01 00:00:00
+
+The value provided to the tag must be in ISO format to be interpreted correctly by the ``!datetime`` tag.
```

Comment:
```suggestion
The value provided to the tag must be in ISO 8601 format to be interpreted correctly by the ``!datetime`` tag.
```
ISO is a vast set of standards, possibly including more than one date/time standard, so specificity might help. Could this maybe also be a hyperlink to https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat, which gives some examples and specifics on Python's implementation?